browser/brave_ads/tabs/ads_tab_helper.h

```diff
@@ -124,7 +126,7 @@ class AdsTabHelper : public content::WebContentsObserver,
   bool was_restored_ = false;
   bool is_new_navigation_ = false;
   std::vector<GURL> redirect_chain_;
-  bool is_error_page_ = false;
+  int http_status_code_ = -1;
```

Comment:
maybe `optional<int>` would be better here?

Reply:
Changed, thanks